database/ffldb/transaction_test.go

```diff
@@ -0,0 +1,564 @@
+package ffldb
+
+import (
+	"bytes"
+	"github.com/kaspanet/kaspad/database"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+func TestKeyValueTransactionCommit(t *testing.T) {
+	// Open a test db
+	path, err := ioutil.TempDir("", "TestKeyValueTransactionCommit")
```

Comment:
Most tests here create a temp database, consider extract function.
Can work similar to DAGSetup - a.k.a. return a cleanup function